wiki/en/Running-a-Server.md

```diff
@@ -79,14 +79,14 @@ To run a Directory [read this guide](Directories)
 
 Most people run Jamulus as a "pure" Server on **hardware without audio** (e.g. on a 3rd party/cloud host) running Linux. The following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution using systemd. To run a server on Windows or on the desktop with a graphical user interface, [see this section](#servers-on-the-desktop).  
 
-If you want to run a Server on a Raspberry Pi (or a different armhf based device), you will need to download the `.deb` files for `armhf`, not the default `amd64` ones you'd use on an Intel/AMD based machine.
+If you want to run a Server on a Raspberry Pi (or a different armhf/arm64 based device), you will need to download the `.deb` files for 32 bit `armhf` or 64 bit `arm64`, not the default `amd64` ones you'd use on an Intel/AMD based machine.
```

Comment:
```
or a different armhf/arm64 based device
```
should probably say something about "debian-based", too.

Reply:
Done.